lib/types.ts

```diff
@@ -241,6 +248,14 @@ export type Target = {
   company?: Pick<Company, 'id'> | Pick<Company, 'name'> | Company;
   date_of_birth?: string;
 };
+export type CredentialsTarget = {
+  first_name: string;
+  last_name: string;
+  social_security_number: string;
+  contact_email: string;
+  date_of_birth: string;
```

Comment:
I left DOB optional in that pydantic update yesterday. Which should it be?

Reply:
required for now until we have screens to collect it